crates/astria-core/Cargo.toml

```diff
@@ -28,6 +28,7 @@ celestia-tendermint = { workspace = true }
 ed25519-consensus = { version = "2.1.0", default-features = false, features = [
   "std",
 ] }
+ethers = { workspace = true }
```

Comment:
I don't think we really want this as a dependency in astria-core? doesn't appear to be used here at all?

Reply:
good catch - this is a remnant of something that was fixed by janis